src/app.ts

```diff
@@ -144,6 +145,7 @@ function createApp(): express.Application {
   app.use(cookieParser());
   app.use(csurf({ cookie: getCSRFCookieOptions(isProduction) }));
 
+  app.use(healthcheckRouter);
```

Comment:
you should be able to add the route in its normal place as the app structure is different here.